### PR TITLE
gsl 2.8

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
- https://gitlab.com/nsnam/ns-3-dev/-/merge_requests/1993

**Comment by chenrui333** [comment]
mostly https://github.com/nest/nest-simulator/issues/3211 issue, we can wait for the upstream to confirm if there was re-tagging.

**Comment by chenrui333** [comment]
still wait for the nest maintainer to confirm about the tagging.

File: .github/autobump.txt [hunk 1]
```diff
@@ -984,6 +984,7 @@ grpc
 grpc-swift
 grype
 gsettings-desktop-schemas
+gsl
```
Comment (carlocab) [inline .github/autobump.txt:987]: This plus `no long build conflict` is preventing other PRs from being merged.

See:
https://github.com/Homebrew/homebrew-core/pull/173471
https://github.com/Homebrew/homebrew-core/pull/173469
https://github.com/Homebrew/homebrew-core/pull/173468

**Review by chenrui333 — APPROVED** [review]

File: .github/autobump.txt (hunk 1 above)
Comment (chenrui333) [reply .github/autobump.txt:987]: indeed, should be good now.

**Comment by chenrui333** [comment]
```
Error: Error: Too many subjects specified. The maximum number of subjects is 64.
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
gnuradio conflict due to #173484

File: Formula/n/ns-3.rb [hunk 2]
```diff
@@ -26,6 +27,12 @@ class Ns3 < Formula
   uses_from_macos "libxml2"
   uses_from_macos "sqlite"
 
+  # cmake patch, upstream PR patch, https://gitlab.com/nsnam/ns-3-dev/-/merge_requests/1993
+  patch do
+    url "https://gitlab.com/nsnam/ns-3-dev/-/commit/08a993d3b48fd219fc3d4a3bf203dc23a910e76a.diff"
+    sha256 "6043985791177f1fc953224db47c96d0b35fd4d38b52a7365ce06826825cebc8"
+  end
+
```
Comment (chenrui333) [inline Formula/n/ns-3.rb:35]: this has been released with ns-3 3.42, https://github.com/Homebrew/homebrew-core/pull/173511

**Comment by chenrui333** [comment]
need another rebase per https://github.com/Homebrew/homebrew-core/pull/174036

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**